docs/www/monitoring.md

```diff
@@ -76,3 +76,31 @@ To opt out of all metrics reporting, set `rpk.enable_usage_stats` to false via
 ```
 rpk config set rpk.enable_usage_stats false
 ```
+
+## Metrics
+
+Through Prometheus, you can access many metrics about the Redpanda process.
+Most of the metrics are used for debugging, but these metrics can be useful to measure system health:
+
+| Metric | Definition | Diagnostics |
+| --- | --- | --- |
+| vectorized_application_uptime | Redpanda uptime in milliseconds |  |
+| vectorized_cluster_partition_last_stable_offset | Last stable offset | If this is the last event received then the cluster is up-to-date and ready for maintenance |
```

Comment:
We do not receive `events` we receive `records` that are grouped by batches. I think it would be more clear to use the same lingo in here. last stable offset is related with read isolation level:
```
This setting controls the visibility of transactional records. Using READ_UNCOMMITTED (isolation_level = 0) makes all records visible. With READ_COMMITTED (isolation_level = 1), non-transactional and COMMITTED transactional records are visible. To be more concrete, READ_COMMITTED returns all data from offsets smaller than the current LSO (last stable offset), and enables the inclusion of the list of aborted transactions in the result, which allows consumers to discard ABORTED transactional records
```